inc/sainexthopgroup.h

```diff
@@ -184,6 +184,18 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_SELECTION_MAP,
 
+    /**
+     * @brief Hierarchical next hop group level.
+     * false: level 2, underlay next hop group
+     * true: level 1, overlay or tunnel next hop group
+     *
+     * @type bool
```

Comment:
As discussed offline, this attribute is not required for most of the other vendors. Please rename the attribute such that it is a control-plane hint which vendor SDKs can choose to ignore.

Reply:
This still needs to be addressed. 

For platforms that do not need this hint, SAI_NEXT_HOP_GROUP_ATTR_IP_NEXTHOP_ONLY will be false, and yet this Nexthop Group may have tunnel nexthops. So this attribute name is misleading.

Reply:
Change the attribute name to generic control plane hint

Reply:
maybe you can change the default value to true. and when it is true, it means the group support h-ecmp (tunnel and ip nexthop). false, means the group only support ip nexthops.

Reply:
Changed the default to true and added more details in comments.
https://github.com/opencomputeproject/SAI/pull/1555/commits/a40e2259e44e77e05d44e9d9412d3200ca4a0001

Reply:
> Changed the default to true and added more details in comments. [a40e225](https://github.com/opencomputeproject/SAI/commit/a40e2259e44e77e05d44e9d9412d3200ca4a0001)

@JaiOCP 

We need to modify the comments as below:

 **false**: Nexthop group consists of **IP nexthop only**
 **true**: Nexthop group consists of **tunnel and IP nexthop**


This is part of [lguohan's comment](https://github.com/opencomputeproject/SAI/pull/1555/files#r1025876735) to make the default as true. "When it is true, it means the group support h-ecmp (tunnel and ip nexthop)."

Reply:
yes, this got messed up in git squash. I have fixed it. Please take a look.
https://github.com/opencomputeproject/SAI/pull/1555/commits/32e7ca6f29da9b3b94d0b21e74dca326414dbc9f

Reply:
There are other next-hop entries other than IP or tunnel a next-hop group can point to, namely MPLS next-hop or SRv6 next-hop.  When this attribute only indicates this is IP+tunnel or IP-only next-hop group, then how to set this attribute correctly in this case?